R/MappedDataBoxplot.R

```diff
@@ -34,6 +36,8 @@ MappedDataBoxplot <- R6::R6Class( # nolint
                           isObserved = TRUE,
                           xlimits = NULL,
                           ylimits = NULL,
+                          xscale = "linear",
+                          yscale = "linear",
```

Comment:
Do we have enums for this? If not: would introduce them (can be done in the next PR).
Compare e.g. https://github.com/Open-Systems-Pharmacology/TLF-Library/blob/ad57bbb909e002823753b634a2e8e063a6f75602/R/utilities-enums.R#L440-L450